gr-digital/python/digital/bindings/crc16_async_bb_python.cc

```diff
@@ -0,0 +1,48 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2021 Cesar Martinez.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ * 
+ */
+
+/***********************************************************************************/
+/* This file is automatically generated using bindtool and can be manually edited  */
```

Comment:
if this file is generated, should it be controlled?

Reply:
Yes. This is how the pybind11 bindings are handled. Probably others can give good reasons why these are in the VCS.

Reply:
This is an unfortunate consequence of our workflow for generating python bindings - since the bindings are scraped from the public headers for each block using a tool called `pygccxml` - 1. this tool is very slow, and 2. for some blocks, the python bindings might not want to expose everything, or do some things differently. 

Perhaps an improvement here would be removing the auto-generated comment for the in-tree bindings.